introduction.tex

```diff
@@ -1,15 +1,31 @@
 \section{Introduction} \label{sec:intro}
 
-The current situation is  that, only the Science Pipelines is released as follows:
+This document collects the needs in therms of releses from different stakehoders, section \ref{sec:reqs}.
+These needs are presented as requirements, that shall not be confused with project requirements, like for example the \gls{DMS} requirement specification, documented in \citeds{LSE-61}.
+Section \ref{sec:process} provides an overview of the release process.
+
+Appedix \ref{sec:defs} provides definitions of the main objects involved in the release activities. 
+Those definitions are given in order to provide a common understanding.
```

Comment:
These definitions provide a common understanding of the components.

Reply:
OK, I would not use the word "components". I better suggest "elements involved"

Reply:
Just say: 
Appendix \ref{sec:defs} provides definitions of the main objects involved in the release activities in order to provide a common understanding.

I don't like introducing a new term